service/hub/server/config.go

```diff
@@ -8,10 +8,23 @@ import (
 	"os"
 	"strings"
 
+	"github.com/cesanta/glog"
 	"github.com/docker/libtrust"
+	"github.com/labring/sealos/pkg/client-go/kubernetes"
 	yaml "gopkg.in/yaml.v2"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+var K8sClient kubernetes.Client
+
+func init() {
```

Comment:
`K8sClient` is only used under same package, so use lowercase var name to prevent variable leak.

Reply:
ok.